lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -5537,14 +5537,6 @@ SwiftASTContext::GetTypeInfo(opaque_compiler_type_t type,
   return swift_flags;
 }
 
-lldb::LanguageType
-SwiftASTContext::GetMinimumLanguage(opaque_compiler_type_t type) {
-  if (!type)
-    return lldb::eLanguageTypeC;
```

Comment:
This is a change in semantic that's pretty substantial -- probably for the best, but I wonder if we're always guaranteed that the type we receive is not null here?

Reply:
Thanks for looking closely! I also thought about this: This was (practically) dead code, since CompilerType::GetMinimumLanguage() implements the same check and thus never calls GetMinimumLanguage with an invalid type. Yes, you could in theory call it directly, but that's not how these methods are meant to be used.

Reply:
I guess we could put in an assertion...